app/assets/stylesheets/dashboard.scss

```diff
@@ -280,11 +280,16 @@ p.debate-status {
 }
 
 #slack-comments {
-  min-height: 160px;
+  .comments {
+    height: 284px;
+  }
 
   .comment {
     padding: 10px;
     background: #fff;
+    -webkit-animation:
```

Comment:
wouldn't this work without prefixing? (or should we also add prefix for firefox?)